app/src/main/assets/queries/nearby_query.rq

```diff
@@ -13,7 +13,7 @@ SELECT
      SERVICE wikibase:around {
        ?item wdt:P625 ?location.
        bd:serviceParam wikibase:center "Point(${LONG} ${LAT})"^^geo:wktLiteral.
-       bd:serviceParam wikibase:radius "${RADIUS}" . # Radius in kilometers.
+       bd:serviceParam wikibase:radius "${RAD}" . # Radius in kilometers.
```

Comment:
I slightly prefer it because it will make code look more aligned.
```
                .replace("${RAD}", String.format(Locale.ROOT, "%.2f", radius))
                .replace("${LAT}", String.format(Locale.ROOT, "%.4f", cur.latitude))
```